app/src/index.tsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./styles/index.scss";
 import App from "./App";
+require("typeface-hind");
+require("typeface-montserrat");
```

Comment:
Why is this using require vs import?

Reply:
no particular reason! i switched to import